app/models/concerns/catkey.rb

```diff
@@ -1,11 +1,50 @@
 # frozen_string_literal: true
 
-module Catkey
+class Catkey
```

Comment:
Should this move out of `app/models/concerns/` now that it's a class?

Reply:
Good idea. I moved it.